Watchman.DomainModel/BotSettings/Queries/Handlers/GetBotVersionQueryHandler.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using Watchman.Cqrs;
+
+namespace Watchman.DomainModel.BotSettings.Queries.Handlers
+{
+    public class GetBotVersionQueryHandler : IQueryHandler<GetBotVersionQuery, GetBotVersionQueryResult>
+    {
+        public GetBotVersionQueryResult Handle(GetBotVersionQuery query)
+        {
+            const string fileName = "version.txt";
```

Comment:
to wrzuciłbym jako field

Reply:
field w tej klasie Hander?

Reply:
![image](https://user-images.githubusercontent.com/35143425/70375828-eae92b80-1902-11ea-8ee3-183a9ce8b64e.png)

Reply:
tak